### ISSUE TITLE
Wrong name in the Arduino Uno rev 3 schematics

### ISSUE THREAD

**Issue by AKJ7** [arduino/Arduino#9173]
In the schematics of the Arduino Uno rev [https://content.arduino.cc/assets/UNOWiFiREV2V4.0_sch.pdf](schematic) at page 7, it is implied that the Controller board uses the Nina W132 wifi module. However the schematics presented to not match those of the from the Nina [https://www.u-blox.com/sites/default/files/NINA-W13_DataSheet_%28UBX-17006694%29.pdf](website) . I believe the right wifi module's name is the NINA W102.

**Comment by FunzioneArduino** [comment]
Yes, hardware team confirmed that the right module's name is NINA W102. The pinout PDF is already up to date, we will fix the schematics asap.
